docs/section22/Rule22-36.md

```diff
@@ -0,0 +1,50 @@
+
+# CHW&CW - Rule 22-36  
+
+**Schema Version:** 0.0.10  
+**Mandatory Rule:** True  
+**Rule ID:** 22-36  
+**Rule Description:** Baseline chilled water system that does not use purchased chilled water shall be modeled with continuous constant flow primary loop and variable flow secondary loop.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**90.1 Section Reference:** Section G3.1.3.10 Chilled-Water Pumps (Systems 7, 8, 11, 12, and 13)  
+**Data Lookup:** None  
+**Evaluation Context:** Building  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, 13b.
+2. B-RMR is modeled with primary-secondary configuration.
+
+**Function Calls:**  
+
+1. get_baseline_system_types()
+2. get_primary_secondary_loops()
+3. get_component_by_id()
+
+**Applicability Checks:**  
+
+- **Check 1:** Get B-RMR system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMR)`
+
+  - Check if B-RMR is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, 13b, i.e. with air-side system served by chiller(s), continue to the next applicability check: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-7", "SYS-8", "SYS-11.1", "SYS-11.2", "SYS-12", "SYS-13", "SYS-7B", "SYS-8B", "SYS-11B", "SYS-12B", "SYS-13B"]): NEXT_APPLICABILITY_CHECK`
+
+  - Else, rule is not applicable to B-RMR: `else: RULE_NOT_APPLICABLE`
+
+- **Check 2:** Get primary and secondary loops for B-RMR: `primary_secondary_loop_dictionary = get_primary_secondary_loops(B_RMR, "COOLING")`
+
+  - Check if B-RMR is modeled with primary secondary configuration, continue to rule logic: `if LEN(primary_secondary_loop_dictionary) != 0: CHECK_RULE_LOGIC`
+
+  - Else, rule is not applicable to B-RMR: `else: RULE_NOT_APPLICABLE`
+
+## Rule Logic:  
+
+- Get primary and secondary loops for B-RMR: `primary_secondary_loop_dictionary = get_primary_secondary_loops(B_RMR, "COOLING")`
+
+  - For each primary loop: `for primary_loop_id in primary_secondary_loop_dictionary.keys():`
+
+    **Rule Assertion - Component:**
+
+    - Case 1: If primary loop is modeled as continuous constant flow and secondary loop as variable flow: `if ( get_component_by_id(primary_loop_id).cooling_or_condensing_design_and_control.flow_control == "FIXED_FLOW" ) AND ( get_component_by_id(primary_loop_id).cooling_or_condensing_design_and_control.operation == "CONTINUOUS" ) AND ( get_component_by_id(secondary_loop_id).cooling_or_condensing_design_and_control.flow_control == "VARIABLE_FLOW" for secondary_loop_id in primary_secondary_loop_dictionary[primary_loop_id] ): PASS`
```

Comment:
maybe we check secondary loops in 22-36 and primary loops in 22-25.

Reply:
22-25 already checks for FIXED FLOW primary, so seems like first condition is redundant so should be removed. 
I also agree with Supriya that checking for CONTINUOUS primary should be moved to 22-25, so that 22-25 deals with primary and 22-36 with secondary. Also, currently CONTINUOUS secondary seems to be missing so should be added.

Reply:
As discussed, we have the following rules:
Pump rules:
22-25 would verify the primary pump is constant speed.
22-10 would verify the secondary pump is constant speed (<300ton) or with VFD (>=300ton).
Loop rules:
22-7 would verify that baseline has been modeled with P S configuration.
22-36 would verify continuous constant flow primary loop and variable-flow secondary loop.
22-34 would verify baseline has been modeled with only one CHW plant (i.e. one primary and one secondary loop).

Reply:
As discussed, we have the following rules:
Pump rules:
22-25 would verify the primary pump is constant speed.
22-10 would verify the secondary pump is constant speed (<300ton) or with VFD (>=300ton).
Loop rules:
22-7 would verify that baseline has been modeled with P S configuration.
22-36 would verify continuous constant flow primary loop and variable-flow secondary loop.
22-34 would verify baseline has been modeled with only one CHW plant (i.e. one primary and one secondary loop).
And I was not able to find CONTINUOUS requirement under Section G3.1.3.10. Please help me confirm.